inc/azure_c_shared_utility/httpapiex.h

```diff
@@ -43,6 +43,25 @@ typedef struct HTTPAPIEX_HANDLE_DATA_TAG* HTTPAPIEX_HANDLE;
 */
 MU_DEFINE_ENUM(HTTPAPIEX_RESULT, HTTPAPIEX_RESULT_VALUES);
 
+/**
+ * @brief    Initialize the HTTPAPIEX.
+ *
+ *  This API shall be called only once before call any other HTTPAPIEX API. 
+ *  **This API is NOT thread safe**.
+ *
+ * @return    An @c HTTPAPIEX_RESULT indicating the status of the call.
+ */
+MOCKABLE_FUNCTION(, HTTPAPIEX_RESULT, HTTPAPIEX_Init);
+
+/**
+ * @brief    Deinitialize the HTTPAPIEX.
+ *
+ *  This API shall be called only once to release all HTTP resources. No other HTTPAPIEX 
+ *      API shall be called after call this API.
+ *  **This API is NOT thread safe**.
+ */
+MOCKABLE_FUNCTION(, void, HTTPAPIEX_Deinit);
```

Comment:
>HTTPAPIEX_Deinit [](start = 26, length = 16)

same.
Generally you create -> initialize -> deinitialize -> delete.

Are these name wrong?

Reply:
This is done all over the place ... please ignore my comment.

---
In reply to: [319617460](https://github.com/Azure/azure-c-shared-utility/pull/366#discussion_r319617460) [](ancestors = 319617460)